test/handle-reply-undefined.test.js

```diff
@@ -0,0 +1,35 @@
+'use strict'
+
+const test = require('ava')
+const { buildApp } = require('./_helper')
+
+test('should handle without logging when context.reply is undefined', async (t) => {
+  t.plan(1)
+
+  const app = buildApp(t)
+
+  app.get('/custom-endpoint', async function () {
+    const query = `query {
+    four: add(x: 2, y: 2)
+    six: add(x: 3, y: 3)
+    echo(msg: "hello")
+    counter
+  }`
+    return app.graphql(query)
```

Comment:
Could you add a test where we use this syntax `app.graphql(query, { reply })`?

I would expect it works in this case